### ISSUE TITLE
[Flaky Test] test_relax_2d_buffer_allocation.py

### ISSUE THREAD

**Issue by tqchen** [apache/tvm#16829]
Seems there are some flaky test cases in hexagon during recent PRs that do not have to do with the feature

### Test(s)

- `tests/python/contrib/test_hexagon/test_relax_2d_buffer_allocation.py::test_alloc_storage_with_scope_global`

### Jenkins Links

- https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-hexagon/detail/PR-16807/5/pipeline
- https://ci.tlcpack.ai/blue/organizations/jenkins/tvm-hexagon/detail/PR-16807/5/pipeline

**Comment by tqchen** [comment]
cc @quic-sanirudh

**Comment by quic-sanirudh** [comment]
Yes, definitely looks like a flaky test, sorry about that. I'll look into this.
